OracleManagementAgent/dockerfiles/latest/container-scripts/common.sh

```diff
@@ -140,7 +140,42 @@ function load_agent_java_options()
   done < "$java_options_file"
 
   log "$APPNAME found java.options [values: $java_options]"
-  AGENT_JAVA_OPTIONS=$(trim "$java_options")
+  AGENT_JAVA_OPTIONS+=$(trim "$java_options")
   export AGENT_JAVA_OPTIONS
   return 0
 }
+
+###########################################################
+# Discover UID and GID values present in this environment
+# IFF username cannot be found then set system property
+# user.name for agent java process in AGENT_JAVA_OPTIONS
+function discover_userinfo()
+{
+  export AGENT_JAVA_OPTIONS=""
+  local uid
+  uid=$(id -u 2>&1)|| true
+  log "UID: $uid"
+
+  local gid
+  gid=$(id -g 2>&1)|| true
+  log "GID: $gid"
+
+  local un_exitcode=0
+  un_output=$(id -un 2>&1)|| un_exitcode=$?
+  if [ $un_exitcode -eq 0 ]; then
+    log "Username is [${un_output}]"
+  else
+    log "INFO: $un_output"
+    log "Please refer to documentation on how to create a named user in container."
```

Comment:
The user inside the container is created by you. It's the user outside the container that needs to match. Also, the phrase "named user" is not broadly used. I would recommend rewording this line.